dotnet/ios/ios13/SupportingMultipleWindowsOniPad/Gallery/View Controllers/GalleryViewController/GalleryCollectionViewCell.cs

```diff
@@ -0,0 +1,38 @@
+﻿/*
+See LICENSE folder for this sample’s licensing information.
+
+Abstract:
+A basic collection view cell that displays an image.
+*/
+
+namespace Gallery;
+public partial class GalleryCollectionViewCell : UICollectionViewCell {
+	public static readonly NSString Key = new NSString (nameof (GalleryCollectionViewCell));
+	public static readonly UINib Nib;
+
+	public UIImage? Image
+	{
+		get
+		{
+			if (imageView is null || imageView.Image is null)
+				return null;
+
+			return imageView.Image;
```

Comment:
Trick here, since you are allows to return a null, there is not need to do the if, simply use the ? operator and return
```suggestion
			return imageView?.Image;
```

And at that point, we an remove the {} and the return and use =>